js/cook.js

```diff
@@ -0,0 +1,205 @@
+            } else if (weight >=40 && weight<= 59) {
+                vDose = 1000+"mg";
+            } else if (weight >=60 && weight<= 90) {
+                vDose = 1500+"mg";
+            } else if (weight >90) {
+                vDose = 2000+"mg";
+            }
+            document.getElementById("stat_dose").innerHTML = "Prescribe a stat dose of " + vDose;
+        }        
+        
+        
+        //Calculate BMI
+        var bmi;
+        function bmi() {
+            var weight = w.value;
+            var height = h.value;
+            var normalweight = document.getElementById('normalweight')
+            
+            if (isNaN(weight)) {
+                bmi = "<em>Weight</em>must be a number!";
+            }else if (isNaN(height)) {
+                bmi = "<em>Height</em>must be a number!";
+            }else if (normalweight.checked) {
+                bmi = "The patient's BMI is normal";
+            }else {
+                bmi = weight/(height/100*height/100);
+        }
+        document.getElementById("bmi_calc").innerHTML = "The patient's BMI is  " + bmi;
+        }
+        
+        // Crcl without gender or creatinine adjustment
+        var CrCl0;
+        
+        function CrCl0() {
+            // validate age, weight, creatinie
+            if (age < 18) {
+                CrCl0 = "<em>Age</em> cannot equal less than 18!";
+            } else if (isNaN(age)) {
+                CrCl0 = "<em>Age</em> must be a number!";
+            } else if (isNaN(weight)) {
+                CrCl0 = "<em>Weight</em> must be a number!";
+            } else if (isNaN(creatinine)) {
+                CrCl0 = "<em>Creatinine</em> must be a number!";
+            } else { 
+                var CrCl0 = ((140-age)*weight);
+            }
+            
+            // output the result (or errors)
+            document.getElementById("Cr_Cl0").innerHTML = CrCl0;
+        }
+        
+        // CrCl with gender adjustment
+        function kidneygender(){
+        var resultG;
+            if (male){
+                resultG = CrCl0 *1.23;
+            } else{
+                resultG = CrCl0 *1.04;
+            }
+            
+            
+            
+            // Weight must be within three months
+        var dVerify;
+        
+        function verifyDate (){
+        var datenear = document.getElementById("date").value;
+        var timestamp = new Date().getTime() - (90 * 24 * 60 * 60 * 1000);
+        
+            if (timestamp < datenear) {
+             dVerify = "Date must be within 3 months";
+            } else {
+             dVerify = "";
+            }
+            document.getElementById("datemessage").innerHTML = dVerify;
+        }
+            
+            
+        // CrCl with creatinine
+        function CrCl(){
+        var creatinine = document.forms["vanc"]["creatine"].value;
+        var CrCl = resultG/creatinine;
+        }
+        
+        
+        //Calculate of BMI
+        var bmi;
```

Comment:
There's another variable called bmi above... if a variable is not global, put it inside the function